examples/plot_fiac_analysis.py

```diff
@@ -0,0 +1,99 @@
+4. A GLM is applied to the dataset (effect/covariance,
+   then contrast estimation)
+"""
+
+print _doc
+
+from os import mkdir, path, getcwd
+
+import numpy as np
+import matplotlib.pyplot as plt
+
+from nilearn.plotting import plot_stat_map
+import nibabel as nib
+
+from nistats.glm import FMRILinearModel
+from nistats.design_matrix import make_design_matrix
+from nistats import datasets
+
+
+# write directory
+write_dir = path.join(getcwd(), 'results')
+if not path.exists(write_dir):
+    mkdir(write_dir)
+
+# Data and analysis parameters
+data = datasets.fetch_fiac_first_level()
+fmri_files = [data['func1'], data['func2']]
+design_files = [data['design_matrix1'], data['design_matrix2']]
+
+# Load all the data into a common GLM
+multi_session_model = FMRILinearModel(fmri_files, design_files, data['mask'])
+
+# GLM fitting
+multi_session_model.fit(do_scaling=True, model='ar1')
+
+
+def make_fiac_contrasts(n_columns):
+    """ Specify some contrasts for the FIAC experiment"""
+    contrast = {}
+    # the design matrices of both runs comprise 13 columns
+    # the first 5 columns of the design matrices correspond to the following
+    # conditions: ['SSt-SSp', 'SSt-DSp', 'DSt-SSp', 'DSt-DSp', 'FirstSt']
+
+    def _pad_vector(contrast_, n_columns):
+        return np.hstack((contrast_, np.zeros(n_columns - len(contrast_))))
+
+    contrast['SStSSp_minus_DStDSp'] = _pad_vector([1, 0, 0, -1], n_columns)
+    contrast['DStDSp_minus_SStSSp'] = - contrast['SStSSp_minus_DStDSp']
+    contrast['DSt_minus_SSt'] = _pad_vector([- 1, - 1, 1, 1], n_columns)
+    contrast['DSp_minus_SSp'] = _pad_vector([- 1, 1, - 1, 1], n_columns)
+    contrast['DSt_minus_SSt_for_DSp'] = _pad_vector([0, - 1, 0, 1], n_columns)
+    contrast['DSp_minus_SSp_for_DSt'] = _pad_vector([0, 0, - 1, 1], n_columns)
+    contrast['Deactivation'] = _pad_vector([- 1, - 1, - 1, - 1, 4], n_columns)
+    contrast['Effects_of_interest'] = np.eye(n_columns)[:5]
+    return contrast
+
+# compute fixed effects of the two runs and compute related images
+n_columns = np.load(design_files[0])['X'].shape[1]
+contrasts = make_fiac_contrasts(n_columns)
+
+print('Computing contrasts...')
+mean_img = multi_session_model.means[0]  # for display
+for index, (contrast_id, contrast_val) in enumerate(contrasts.items()):
+    print('  Contrast % 2i out of %i: %s' % (
+        index + 1, len(contrasts), contrast_id))
+    z_image_path = path.join(write_dir, '%s_z_map.nii' % contrast_id)
+    z_map, = multi_session_model.contrast(
+        [contrast_val] * 2, con_id=contrast_id, output_z=True)
+    nib.save(z_map, z_image_path)
+
+    # make a snapshot of the contrast activation
+    if contrast_id == 'Effects_of_interest':
+        vmax = max(- z_map.get_data().min(), z_map.get_data().max())
+        vmin = - vmax
+        plot_stat_map(z_map, bg_img=mean_img, threshold=2.5, title=contrast_id)
+        plt.savefig(path.join(write_dir, '%s_z_map.png' % contrast_id))
```

Comment:
Rather than using plt.savefig, I guess the "nilearn way" would be to do:

```python

display = plot_stat_map(z_map, bg_img=mean_img, threshold=2.5, title=contrast_id)
display.savefig(path.join(write_dir, '%s_z_map.png' % contrast_id))
```